channels/api.py

```diff
@@ -506,6 +506,59 @@ def create_channel_groups_and_roles(channel):
     return roles
 
 
+def comment_values_from_reddit(comment):
+    """
+    Returns the values to populate the comment from a comment
+
+    Args:
+        comment (praw.models.Comment): the reddit comment to source data from
+
+    Returns:
+        dict: property values for a channels.models.Comment record
+    """
+    _, parent_id = get_kind_and_id(comment.parent_id)
+    return dict(
+        parent_id=parent_id,
+        text=comment.body,
+        score=comment.score,
+        edited=comment.edited if comment.edited is False else True,
+        removed=comment.banned_by is not None,
+        deleted=comment.body == DELETED_COMMENT_OR_POST_TEXT,
+        created_on=datetime.fromtimestamp(comment.created, tz=timezone.utc).isoformat(),
+    )
+
+
+def create_comment(post, comment, author):
```

Comment:
Can we change to `create_comment(*, post, comment, author)` to require keyword args for clarity?

Reply:
I'm not sure I understand why they need to be keyword args?